lib/portlayer/storage/vsphere/image.go

```diff
@@ -70,7 +71,16 @@ func NewImageStore(ctx context.Context, s *session.Session) (*ImageStore, error)
 	// Currently using the datastore associated with the session which is not
```

Comment:
Might as well update this comment since after this change it's no longer relevant.